compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2804,14 +2804,14 @@ TR::Register *OMR::Power::TreeEvaluator::vicmpleEvaluator(TR::Node *node, TR::Co
    return TR::TreeEvaluator::vicmpgeEvaluator(node, cg);
    }
 
-TR::Register *OMR::Power::TreeEvaluator::vselectEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+TR::Register *OMR::Power::TreeEvaluator::vbitselectEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::inlineVectorTernaryOp(node, cg, TR::InstOpCode::vsel);
+   return TR::TreeEvaluator::inlineVectorSelectOp(node, cg, TR::InstOpCode::vsel);
```

Comment:
I believe this function should be `inlineVectorBitSelectOp` since the opcode was renamed.  This comment applies elsewhere to the other evaluators that call it too.